adafruit_mpu6050.py

```diff
@@ -176,6 +204,46 @@ class MPU6050:
             gyro_x, gyro_y, gyro_z = sensor.gyro
             temperature = sensor.temperature
 
+    === Test Script ===
+    # What I used to get everything going
+    #import busio
+    #i2c =  busio.I2C(scl=board.A1, sda=board.A0)
+    #imu = MPU6050(i2c, 0x69)
+
+    # What I presume testers will use
+    import board
+    i2c = board.I2C()
+    imu = (i2c)
+
+
+    # Test Class value assignments
+    print("Test of Class assigments")
+    imu.clock_source = Clksel.CLKSEL_INTERNAL_8MHz
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source = Clksel.CLKSEL_INTERNAL_X
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source = Clksel.CLKSEL_INTERNAL_Y
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source = Clksel.CLKSEL_INTERNAL_Z
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source = Clksel.CLKSEL_EXTERNAL_32
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source =  Clksel.CLKSEL_EXTERNAL_19
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source = Clksel.CLKSEL_RESERVED
+    print("Clock source is: ", imu.clock_source)
+    imu.clock_source = Clksel.CLKSEL_STOP
+    print("Clock source is: ", imu.clock_source)
+
+    # Test integer Assignments
+    print("Test Integer assignments")
+    for x in range(0,8):
+        imu.clock_source=x
+        print("Clock source is: ", imu.clock_source)
+
+    # Produce an error - comment out as needed
+    imu.clock_source = -1
+    imu.clock_source = 8
```

Comment:
Thanks for explaining!  It looks like this is giving the CI a hard time, and while its helpful we don't need to have it explicitly in the library.  Would you mind removing it?  Thanks!  I'll take another look once this passes the CI.

Reply:
On it.  If it doesn't pass... I'll recheck again in the AM.